src/index.html

```diff
@@ -22,6 +22,536 @@
+        />
+
+        <div class="product-card__product-details">
+          <h3 class="product-card__product-details-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="product-card__product-details-description">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+          </div>
+
+          <p class="product-card__rating-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+
+          <p class="product-card__price-cost">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="product-card__button-buy"
+        >
+          buy
+        </a>
+      </section>
+
+      <!-- card #8 -->
+      <section class="product-card">
+        <img
+          class="product-card__product-image"
+          src="./images/imac.jpeg"
+          alt="apple-imac-27"
+        />
+
+        <div class="product-card__product-details">
+          <h3 class="product-card__product-details-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="product-card__product-details-description">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+          </div>
+
+          <p class="product-card__rating-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+
+          <p class="product-card__price-cost">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="product-card__button-buy"
+        >
+          buy
+        </a>
```

Comment:
The product card section is repeated multiple times with the same content. This violates the DRY (Don't Repeat Yourself) principle. Consider using a loop in a templating language or a JavaScript framework to generate these sections dynamically.